testData/deviceReporter/acceptanceTests/chrome.json

```diff
@@ -1,5 +1,9 @@
 [
     {
         "id": "com.ilunion.cloud4chrome"
```

Comment:
@kaspermarkus what about org.chrome.cloud4chrome and org.mozilla.cloud4firefox. Should those also be removed?